using_images/sti_images/ruby.adoc

```diff
@@ -10,10 +10,18 @@
 toc::[]
 
 == Overview
-OpenShift provides https://github.com/openshift/source-to-image[STI] enabled Ruby images for building and running Ruby applications.  These images can be used with the link:../../architecture/core_objects/builds.html#sti-build[STI build strategy].
+OpenShift provides https://github.com/openshift/source-to-image[STI] enabled 
+Ruby images for building and running Ruby applications. These images can be 
+used with the 
+link:../../architecture/core_objects/builds.html#sti-build[STI build strategy];
+the https://github.com/openshift/sti-ruby[STI builder image] assembles your
+application source with any required dependencies to create a new image
+containing your Ruby application. This resulting image can be run either by
+OpenShift or by Docker.
 
 == Versions
-Currently, OpenShift only provides version https://github.com/openshift/sti-ruby/tree/master/2.0[2.0] of Ruby.
+Currently, OpenShift only provides version 
```

Comment:
here and everywhere else we should strike out "only" and just say "currently openshift providers version......"